WordPress/Helpers/WPGlobalVariablesHelper.php

```diff
@@ -0,0 +1,313 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace WordPressCS\WordPress\Helpers;
+
+/**
+ * Helper utilities for recognizing WP global variables.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @since   3.0.0 The property in this class was previously contained in the
+ *                `WordPressCS\WordPress\Sniff` class and has been moved here.
+ */
+final class WPGlobalVariablesHelper {
+
+	/**
+	 * List of global WP variables.
+	 *
+	 * @since 0.3.0
+	 * @since 0.11.0 Changed visibility from public to protected.
+	 * @since 0.12.0 Renamed from `$globals` to `$wp_globals` to be more descriptive.
+	 * @since 0.12.0 Moved to the `Sniff` class from the WordPress.Variables.GlobalVariables sniff.
+	 * @since 3.0.0  - Moved from the Sniff class to this class.
+	 *               - The property visibility has changed from `protected` to `private static`.
+	 *                 Use the `get_names()` method for access or use the `is_wp_global()` method
+	 *                 to check if a new is on the list.
```

Comment:
```suggestion
	 *                 to check if a name is on the list.
```

Oops - just noticed this typo. Fixed now.